### DISCUSSION TITLE
[vcpkg] Have a list of all installed files for all ports and triplets somewhere

### DISCUSSION THREAD

**Discussion by Neumann-A** [microsoft/vcpkg#17823] [Ideas]
Sometimes having a list of all filenames would make fixing some things easier. Especially if you do not have access to a certain platform.

**Comment by cbezault** [comment]
We must already keep a list like this in memory (at least for all files in the install directory) as I distinctly remember making a change which prints out what port a file you're trying to install conflicts with.

**Comment by Neumann-A** [comment]
@cbezault: there is `<vcpkgroot>/installed/info` with all the `*.list` files. Maybe the Baseline CI can publish a compressed file of that folder somewhere for every triplet?
